collect_app/src/main/res/layout/blank_form_list_item.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please run auto-format on this to reorder attributes

Reply:
Done.